From e111b9131d61c7b4e1bfa7690fe4b73311daad56 Mon Sep 17 00:00:00 2001
From: "F. von Gellhorn" <flinux@vongellhorn.ch>
Date: Sun, 18 Oct 2020 23:40:05 +0200
Subject: [PATCH 1/1] Solus tweaks to the default config

- switch to launchermenu
- Move miniizeall between systemtray and the clock
---
 .../contents/layout.js                                   | 9 +++++----
 1 file changed, 5 insertions(+), 4 deletions(-)

diff --git a/layout-templates/org.kde.plasma.desktop.defaultPanel/contents/layout.js b/layout-templates/org.kde.plasma.desktop.defaultPanel/contents/layout.js
index e2af84c..d48868f 100644
--- a/layout-templates/org.kde.plasma.desktop.defaultPanel/contents/layout.js
+++ b/layout-templates/org.kde.plasma.desktop.defaultPanel/contents/layout.js
@@ -42,9 +42,9 @@ if (panel.formFactor === "horizontal") {
     }
 }
 
-var kickoff = panel.addWidget("org.kde.plasma.kickoff")
-kickoff.currentConfigGroup = ["Shortcuts"]
-kickoff.writeConfig("global", "Alt+F1")
+var kicker = panel.addWidget("org.kde.plasma.kicker")
+kicker.currentConfigGroup = ["Shortcuts"]
+kicker.writeConfig("global", "Alt+F1")
 
 //panel.addWidget("org.kde.plasma.showActivityManager")
 panel.addWidget("org.kde.plasma.pager")
@@ -94,6 +94,7 @@ if (langIds.indexOf(languageId) != -1) {
 }
 
 panel.addWidget("org.kde.plasma.systemtray")
+panel.addWidget("org.kde.plasma.minimizeall")
 panel.addWidget("org.kde.plasma.digitalclock")
-panel.addWidget("org.kde.plasma.showdesktop")
+
 
-- 
2.28.0

